test/nodes/test_file_converter.py

```diff
@@ -142,6 +148,42 @@ def test_page_range_numbers(Converter):
     assert documents[1].meta["page"] == 4
 
 
+@pytest.mark.unit
+@pytest.mark.parametrize("Converter", [PDFToTextConverter])
+def test_pdf_parallel(Converter):
+    converter = Converter(multiprocessing=True)
+    document = converter.convert(file_path=SAMPLES_PATH / "pdf" / "sample_pdf_6.pdf")[0]
+
+    pages = document.content.split("\f")
+
+    assert pages[0] == "This is the page 1 of the document."
+    assert pages[-1] == "This is the page 500 of the document."
+
+
+@pytest.mark.unit
+@pytest.mark.parametrize("Converter", [PDFToTextConverter])
+def test_pdf_parallel_page_range(Converter):
+    converter = Converter(multiprocessing=True)
+    document = converter.convert(file_path=SAMPLES_PATH / "pdf" / "sample_pdf_6.pdf", start_page=2)[0]
+
+    pages = document.content.split("\f")
+
+    assert pages[0] == ""
+    assert len(pages) == 500
+
+
+@pytest.mark.unit
+@pytest.mark.parametrize("Converter", [PDFToTextConverter])
+def test_pdf_parallel_layout(Converter):
+    converter = Converter(multiprocessing=True, keep_physical_layout=True)
+    document = converter.convert(file_path=SAMPLES_PATH / "pdf" / "sample_pdf_6.pdf")[0]
+
+    pages = document.content.split("\f")
+
+    assert pages[0] == "This is the page 1 of the document."
+    assert pages[-1] == "This is the page 500 of the document."
+
```

Comment:
Let's add the following tests here to be reminded to remove the deprecated parameter. For these tests we need to import `fail_at_version` from `..conftest`.
```suggestion


@fail_at_version(1, 17)
def test_deprecated_encoding():
	with pytest.warns(DeprecationWarning):
		converter = PDFToTextConverter(encoding="utf-8")


@fail_at_version(1, 17)
def test_deprecated_encoding_in_convert_method():
	converter = PDFToTextConverter()
	with pytest.warns(DeprecationWarning):
		converter.convert(file_path=SAMPLES_PATH / "pdf" / "sample_pdf_1.pdf", encoding="utf-8")


@fail_at_version(1, 17)
def test_deprecated_keep_physical_layout():
	with pytest.warns(DeprecationWarning):
		converter = PDFToTextConverter(keep_physical_layout=True)


@fail_at_version(1, 17)
def test_deprecated_keep_physical_layout_in_convert_method():
	converter = PDFToTextConverter()
	with pytest.warns(DeprecationWarning):
		converter.convert(file_path=SAMPLES_PATH / "pdf" / "sample_pdf_1.pdf", keep_physical_layout=True)
```

Reply:
Sorry, I forgot the deprecation tests that you first mentioned. They were not on the second iteration.